eofconvention.fsx

```diff
@@ -0,0 +1,29 @@
+open System.IO
+
+let EOFIsEOL(path: FileInfo) = 
```

Comment:
@realmarv I'd prefer a better name for this function, such as EolAtEof

Reply:
Done